ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2196,6 +2196,12 @@ sflow:
     < sflow_destination_ip_2 >:
   source_interface: < source_interface >
   run: < true | false >
+  hardware_acceleration:
+    enabled: < true | false >
+    sample: < sample_rate >
+    modules:
+      - name: < module name >
+        enabled: < true | false >
```

Comment:
As per the logic in cli, the default value is true.

Reply:
Added info that default is true.